doc/src/user_guide.rst

```diff
@@ -712,1034 +570,997 @@ Example::
+the mesh generation stage. Two faces may be taged with
+``periodic_l_x`` and ``periodic_r_x``, where ``x`` is a unique
+integer for the pair of boundaries. Currently, only periodicity in a
+single cardinal direction is supported, for example, the planes
+:math:`(x,y,0)` and :math:`(x,y,10)`. 
 
-    *float*
+[soln-ics]
+^^^^^^^^^^
 
-2. ``basedir`` --- relative path to directory where outputs will be
-   written:
+Parameterises space (x, y, [z]) dependent initial conditions with
+
+1. ``rho`` --- initial density distribution for ``euler`` |
+   ``navier-stokes``:
 
     *string*
 
-3. ``basename`` --- pattern of output names:
+2. ``u`` --- initial x-velocity distribution for ``euler`` |
+   ``navier-stokes`` | ``ac-euler`` | ``ac-navier-stokes``:
 
     *string*
 
-4. ``post-action`` --- command to execute after writing the file:
+3. ``v`` --- initial y-velocity distribution for ``euler`` |
+   ``navier-stokes`` | ``ac-euler`` | ``ac-navier-stokes``:
 
     *string*
 
-5. ``post-action-mode`` --- how the post-action command should be
-   executed:
-
-    ``blocking`` | ``non-blocking``
-
-4. ``region`` --- region to be written, specified as either the
-   entire domain using ``*``, a cuboidal sub-region via diametrically
-   opposite vertices, or a sub-region of elements that have faces on a
-   specific domain boundary via the name of the domain boundary
-
-    ``*`` | ``[(x, y, [z]), (x, y, [z])]`` | *string*
-
-Example::
-
-    [soln-plugin-writer]
-    dt-out = 0.01
-    basedir = .
-    basename = files-{t:.2f}
-    post-action = echo "Wrote file {soln} at time {t} for mesh {mesh}."
-    post-action-mode = blocking
-    region = [(-5, -5, -5), (5, 5, 5)]
-
-[soln-plugin-fluidforce-*name*]
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Periodically integrates the pressure and viscous stress on the boundary
-labelled ``name`` and writes out the resulting force vectors to a CSV
-file. Parameterised with
-
-1. ``nsteps`` --- integrate every ``nsteps``:
-
-    *int*
-
-2. ``file`` --- output file path; should the file already exist it
-   will be appended to:
+4. ``w`` --- initial z-velocity distribution for ``euler`` |
+   ``navier-stokes`` | ``ac-euler`` | ``ac-navier-stokes``:
 
     *string*
 
-3. ``header`` --- if to output a header row or not:
+5. ``p`` --- initial static pressure distribution for ``euler`` |
+   ``navier-stokes`` | ``ac-euler`` | ``ac-navier-stokes``:
 
-    *boolean*
+    *string*
 
 Example::
 
-    [soln-plugin-fluidforce-wing]
-    nsteps = 10
-    file = wing-forces.csv
-    header = true
-
-[soln-plugin-nancheck]
-^^^^^^^^^^^^^^^^^^^^^^
-
-Periodically checks the solution for NaN values. Parameterised with
+    [soln-ics]
+    rho = 1.0
+    u = x*y*sin(y)
+    v = z
+    w = 1.0
+    p = 1.0/(1.0+x)
 
-1. ``nsteps`` --- check every ``nsteps``:
+Nodal Point Sets
+----------------
 
-    *int*
+Flux reconstruction is a high-order nodal spectral element method, and
```

Comment:
I would replace with something shorter, like: "Solution point sets must be specified for each element type that is used and flux point sets must be specified for each interface type that is used. If anti-aliasing is enabled then quadrature point for each element and interface type that is used must also be specified."